code/datums/mood_events/generic_negative_events.dm

```diff
@@ -113,7 +113,7 @@
 
 /datum/mood_event/brain_damage/add_effects()
 	var/damage_message = pick_list_replacements(BRAIN_DAMAGE_FILE, "brain_damage")
```

Comment:
Remove this if you're not going to use it.

Reply:
oopsie doopsies ill do that thank you